cfme/middleware/__init__.py

```diff
@@ -1,8 +1,39 @@
-from cfme.web_ui import CheckboxTable, toolbar as tb
 from functools import partial
 
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import CheckboxTable, toolbar as tb
+from utils.browser import ensure_browser_open
+from cfme.common import Validatable, SummaryMixin
+from utils.log import logger
+
+
 cfg_btn = partial(tb.select, 'Configuration')
 mon_btn = partial(tb.select, 'Monitoring')
 pol_btn = partial(tb.select, 'Policy')
+pwr_btn = partial(tb.select, 'Power')
+
+
+def list_tbl():
+    return CheckboxTable(table_locator="//div[@id='main-content']//div[@id='list_grid']/table")
```

Comment:
@mfalesni kept only the `LIST_TABLE_LOCATOR = "//div[@id='list_grid']/table"` as common. Created local copy for each module.